roseau/load_flow/_solvers.py

```diff
@@ -162,9 +163,24 @@ def save_matrix(self, prefix: str) -> None:
         """
         raise NotImplementedError(f"save_matrix() is not implemented for solver {self.name!r}.")
 
-    def current_jacobian(self) -> FloatMatrix:
+    def jacobian(self) -> FloatMatrix:
```

Comment:
Would this be better as a property similar to other properties?

Reply:
The difference is that this method can be really slow on large networks as it is computing a non-sparse version of the jacobian (with complexity O(n²)). I don't know if it is specific to my debugger, but it tends to compute all the properties when looking at an object so I didn't want to have a slowdown for this. Anyway I have a feeling that properties shouldn't be used for something not immediate that requires heavy computations.